### DISCUSSION TITLE
UTF-8 Encoding Issue with PostgreSQL in DBeaver

### DISCUSSION THREAD

**Discussion by matheus-kas** [dbeaver/dbeaver#20083] [Q&A]
I'm using DBeaver with PostgreSQL and I'm facing an issue. Every time I run a SELECT query, I get the following error:

SQL Error [22021]: ERROR: invalid byte sequence for encoding "UTF8": 0xc7 0xc3
My database is in UTF-8 and, as far as I know, so is the data. However, I still encounter this error and it's causing difficulty in accessing my data.

Do you have any step-by-step guide to resolve this? Could the data possibly contain non-UTF8 characters? How can I handle this in DBeaver?

Thank you in advance for your help!

Best regards,

**Comment by LonwoLonwo** [comment]
You can vote for this issue: https://github.com/dbeaver/dbeaver/issues/11139
For now, we do not have answers for this problem, unfortutanely.
